app/src/docker/docker-compose.yml

```diff
@@ -153,6 +154,24 @@ services:
       - internal
     restart: on-failure:5
 
+  assess-claim-dc6602v2:
```

Comment:
Let's not add this here for a while. There is a docker-compose extension files. I think that would be better at this stage.

Reply:
Like this? https://docs.docker.com/compose/extends/